### PR TITLE
chore(node version): update node version

### PR DESCRIPTION
**Pull request by mansurissa**
#### Description of Task to be completed?
- update app to use node 12.x and above
- added envVar of develop
- update dev script to use seed:undo
- uninstalling unused packaes
#### How should this be manually tested?
1.  clone this repo
1. Make sure you are using the latest version on node and
1.  create a `.env` file and copy all variables from `.env.example`
1. run `yarn install`
1.  go to `http://localhost:3000/` 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[]()
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by ceelogre — COMMENTED** [review]
Is this a new `.env` variable?
Capture d’écran 2021-09-18 à 21 03 32

**Comment by mansurissa** [comment]
> Is this a new `.env` variable?
> Capture d’écran 2021-09-18 à 21 03 32

No, I think you need todo migrate undo and migrate again

**Comment by ceelogre** [comment]
Remind me how to deal with unfound migrations, I've dropped the 

Capture d’écran 2021-09-19 à 09 07 32

**Comment by mansurissa** [comment]
> Remind me how to deal with unfound migrations, I've dropped the
> 
> Capture d’écran 2021-09-19 à 09 07 32

Did you delete it?

**Comment by ceelogre** [comment]
Not really, can't figure out what happened! I've also tried deleting the `sequelizemeta` table but the error persists.

**Comment by ceelogre** [comment]
I've solved the above issue however, after migrating, the error `column "googleId" of relation "users" does not exist` is still there.

**Comment by mansurissa** [comment]
> I've solved the above issue however, after migrating, the error `column "googleId" of relation "users" does not exist` is still there.

Can you guide me so that I can reproduce the error? cause it working fine on my side(migrations).

**Review by ceelogre — APPROVED** [review]
I've fixed the issue. It's working as expected.

**Review by RedJanvier — APPROVED** [review]
LGTM!

File: src/database/config.js [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-      }
-
-      else 
-      {
-        console.log(`Connecting to ${envVar} db`)
-        environment = env[envVar]
-      }
-        return environment
+  let environment = {};
+  if (envVar === 'docker') {
```
Comment (RedJanvier) [inline src/database/config.js:7]: This could also support the `development` environment too!

**Review by RedJanvier — COMMENTED** [review]
Is it possible to add development alongside with the docker environment?

**Comment by mansurissa** [comment]
> Is it possible to add development alongside with the docker environment?

There is `if('docker'){----}else if('production){---}else{and this where we assume to be development}` I had added that but @denislohan told explained to me that logic after.

**Pull request merged**